REFRAME.md

```diff
@@ -94,6 +137,26 @@ type Response union {
 Note: Each Request type has a corresponding Response type.
 Every message except the Error type should end in Request/Response.
 
+#### Cachable/Non-Cachable Methods
+
+The following methods are cachable:
+
+```ipldsch
+type Request union {
+    | "IdentifyRequest" IdentifyRequest
+    | "FindProvidersRequest" FindProvidersRequest
+    | "GetIPNSRequest" GetIPNSRequest
+}
+```
+
+The following methods are non-cachable:
+
+```ipldsch
+type Request union {
+    | "PutIPNSRequest" PutIPNSRequest
+}
+```
+

```

Comment:
@lidel I added this to describe the indicators for transports on cacheable/non-cacheable to support HTTP. I could tag the individual methods, but this seemed perhaps easier to look at. Happy to tag the methods individually though.

Reply:
> Implementations are encouraged to improve performance of  `CachableRequest` methods by applying transport-specific caching strategies.

Transport and method-specific caching strategies, right?